src/lib/cda/cda.js

```diff
@@ -0,0 +1,37 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+let parseString = require('xml2js').parseString;
+let dataHandler = require('../dataHandler/dataHandler');
+
+module.exports = class cda extends dataHandler {
+    constructor(dataType) {
+        super(dataType);
```

Comment:
Should this fill in the dataType to 'cda' instead of taking it as a constructor argument?

Reply:
good catch